lms/static/sass/discussion/_discussion.scss

```diff
@@ -143,15 +143,15 @@ body.discussion {
   }
 
   .wmd-button {
-    @include padding-right(3px);
-    @include padding-left(3px);
     position: absolute;
     display: inline-block;
     width: 20px;
     height: 20px;
     background: none;
     list-style: none;
     cursor: pointer;
+    border: none;
```

Comment:
Hm, yeah, weird. Talbs always out "out to in", so border would be beyond the background, and even padding.